### ISSUE TITLE
License

### ISSUE THREAD

**Issue by sarah-johnson** [karst-club/karst#34]
Add a license to the project.

We talked about adopting http://www.opengamingfoundation.org/ogl.html which also has an addendum for free/open culture but also not commercialization

**Comment by vonkow** [comment]
So, the overall spirit that I'd like to take is one where: 
* the core rules, setting, and tools are be free and free to expand upon for all 
* we could theoretically create "official" Karst things that may or may not be entirely free
* others could create things "for use with" Karst that may or may not be entirely free

Philosophically speaking, this would protect the game from someone else "stealing" it and monetizing it, but would otherwise let people do what they want with it. If we ever wanted to sell printed copies with commissioned artwork or if someone else wanted to write and sell an adventure module or whatever, they would be free to.

As far as implementation, I think this would involve:
* The OGL or similar for the rules with a note about Product Identity covering the name and setting
* A 3rd Party reuse license for the aforementioned Product Identity that allows others to use it for their own creative purposes. There are a number of these out there, I will do more research on them.
* Some kind of Open Source license for the code, maybe MIT?

Also very happy to consider other ideas about licensing.

**Comment by vonkow** [comment]
A couple things.

I still need to add in the OGL and MIT license, assuming we're cool with the MIT license.

Would love feedback on the 3rd-party license. I modeled it after some other 3rd-party licenses, but with less restrictions.

Still have to figure out what entity to assign the copyright to. Some options include:
* Yours truly
* Dua (formerly Duatronix), a fictional organization that I have occasionally used as an entity in the past. The name comes from a very loud kitten that I lived with in the woods (Dua being short for Cadua, or "tail" in latin).
* Karst Club
* ???
